### ISSUE TITLE
Complex motor stereotypies

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7314]
**Preferred term label:**
Complex motor stereotypies

**Synonyms**

**Definition (free text, please give PubMed ID)**
Motor stereotypies are repetitive, rhythmic, often bilateral movements with a fixed pattern (e.g.,hand flapping, waving, or rotating) and regular frequency that can usually be stopped by distraction. PMID: 28405185

Complex motor stereotypes include handshaking, posturing, flapping or waving, opening and closing of the hands, finger writhing, arm flapping, and flexion and extension of the wrists. Additional movement patterns may occur in conjunction (e.g. body rocking, leg shaking or kicking, facial grimacing, mouth opening, neck extension, and involuntary noises), but the hand/arm movements are dominant. PMID: 1953203

**Parent term (use hpo.jax.org/app)**
Motor Stereotypies (Term does not currently exist.)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
-- requires discussion together with echophenomenon

**Comment by pnrobinson** [comment]
@bcoleman96 
Some of the children of Motor stereotypy/repetitive mannerisms are complex. I am thinking that nearly every movement can be considered complex and so I do not think this term really adds much, I will close for now, but reopen if needed.
